org.eclipse.scanning.device.ui/src/org/eclipse/scanning/device/ui/points/ExecuteView.java

```diff
@@ -331,6 +333,7 @@ private void showQueue() {
 	 * @throws Exception
 			// TODO Use IScanBuilderService
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)